data-table/table.js

```diff
@@ -666,4 +666,6 @@ function dtCreateDataTable(clientConfig) {
 /* eslint no-undef: ["off"] */
 if (typeof exports !== typeof undefined) {
     exports.createDataTable = dtCreateDataTable;
+    exports.configDict = configDict;
+    exports.validateConfig = validateConfig;
```

Comment:
I'm not convinced either of these should be exported - by exposing them, you're indicating to real clients that these are functions they should use (rather than purely internal functions). Can you get this done without exporting them?

For example, to test `numRows`, you could actually look at the HTML table and count the number of rows in it with javascript.